commcare-salesforce-jobs/MOH513-Update-Household-in-SF-Production.js

```diff
@@ -28,7 +31,7 @@ upsert("Household__c","CommCare_Code__c",fields(
 
 upsert("Visit__c","CommCare_Visit_ID__c", fields(
   field("CommCare_Visit_ID__c", dataValue("id")),
-  relationship("Household__r","CommCare_Code__c",dataValue("form.case.@case_id")),
+  relationship("Household__r","CommCare___c",dataValue("form.case.@case_id")),
```

Comment:
@juliahiggins Is `CommCare__c` a new field on Household? If yes, please be sure (1) this is set as an External Id field in SF (compare this field's configuration with the old field), and (2) make sure this field is populated across all records. 

Context: Previously, we were using `CommCare_Code__c` (a field with the "External Id" setting enabled) on Household to look up the matching parent record.

Reply:
@aleksa-krolls Sorry, this must have been a typo on my part while pasting in the code from the old Update HH job - we haven't added a new field here and the mapping to CommCare_Code__c should remain constant!